projects/go-lib/src/lib/components/go-datepicker/go-calendar.component.ts

```diff
@@ -63,15 +70,11 @@ export class GoCalendarComponent implements OnDestroy, OnInit {
   }
 
   ngOnInit(): void {
-    this.subscription = this.calendar.calendarOpen.subscribe((value: boolean) => {
-      if (value) {
         this.selectedDate = this.calendar.selectedDate;
         this.initializeDate();
-      }
-      this.opened = value;
-    });
-
-    this.dateAdapter.setLocale(this.locale);
+        this.appendToCalender();
+        this.opened = true;
+        this.dateAdapter.setLocale(this.locale);
   }
 
   ngOnDestroy(): void {
```

Comment:
In this method, you'll want to add this method call above `this.closeCalendar();`. New method should look like:
```ts
ngOnDestroy(): void {
  this.removeCalenderfromBody();
  this.closeCalendar();
}
```